scripts/data_flow/sync_field_data.py

```diff
@@ -0,0 +1,320 @@
+#! /usr/bin/env python3
+
+###############################################################################
+# Script monitors ingest directories for newly written files and then syncs
+# the file to the appropriate directory based on the file type.
+#
+# This script runs from cron on eol-rosetta as user ads.
+# The crontab expects the script to be in /net/jlocal/projects/scripts.
+#
+###############################################################################
+
+import logging
+import logging.handlers
+import argparse
+import os
+import sys
+import smtplib
+from email.mime.text import MIMEText
+
+# set up variables
+# Have to get PROJ_DIR, project and aircraft from env here even though will
+# get them again when import fieldProc_setup, because need them to find
+# location of fieldPro_setup.
+
+full_proj_dir = os.environ['PROJ_DIR'] + '/' + os.environ['PROJECT'] + '/' + os.environ['AIRCRAFT']+'/'
```

Comment:
@sararunkel Once you resolve how to gracefully catch if an env var isn't set in your test suite, please add that same error catching here. Right now, if AIRCRAFT isn't set, the code crashes:

>  $ ./sync_field_data.py 
> Traceback (most recent call last):
>   File "/Users/janine/dev/aircraft_projects/scripts/data_flow/./sync_field_data.py", line 25, in <module>
>     full_proj_dir = os.environ['PROJ_DIR'] + '/' + os.environ['PROJECT'] + '/' + os.environ['AIRCRAFT']+'/'
>                                                                                  ~~~~~~~~~~^^^^^^^^^^^^
>   File "<frozen os>", line 714, in __getitem__
> KeyError: 'AIRCRAFT'
>

Reply:
Added in the same check_env.py method to sync_field_data.py